recipes/dpath/meta.yaml

```diff
@@ -0,0 +1,107 @@
+    \ afilter(x):\n    ...     if \"ffle\" in str(x):\n    ...             return True\n    ...     return False\n    ...\n    >>> result = dpath.util.search(x, '**', afilter=afilter)\n    >>> print json.dumps(result,\
+    \ indent=4, sort_keys=True)\n    {\n    \"a\": {\n        \"b\": {\n        \"c\": \"Waffles\",\n        \"d\": \"Waffles\",\n        \"e\": {\n            \"f\": {\n            \"g\": \"Roffle\"\n\
+    \            }\n        }\n        }\n    }\n    }\n\nObviously filtering functions can perform more advanced tests (regular\nexpressions, etc etc).\n\nKey Names\n=========\n\nBy default, dpath only\
+    \ understands dictionary keys that are integers or\nstrings. String keys must be non-empty. You can change this behavior by\nsetting a library-wide dpath option:\n\n.. code-block:: python\n\n    import\
+    \ dpath.options\n    dpath.options.ALLOW_EMPTY_STRING_KEYS = True\n\nAgain, by default, this behavior is OFF, and empty string keys will\nresult in ``dpath.exceptions.InvalidKeyName`` being thrown.\n\
+    \nSeparator got you down? Use lists as paths\n==========================================\n\nThe default behavior in dpath is to assume that the path given is a string, which must be tokenized by splitting\
+    \ at the separator to yield a distinct set of path components against which dictionary keys can be individually glob tested. However, this presents a problem when you want to use paths that have a separator\
+    \ in their name; the tokenizer cannot properly understand what you mean by '/a/b/c' if it is possible for '/' to exist as a valid character in a key name.\n\nTo get around this, you can sidestep the\
+    \ whole \"filesystem path\" style, and abandon the separator entirely, by using lists as paths. All of the methods in dpath.util.* support the use of a list instead of a string as a path. So for example:\n\
+    \n.. code-block: python\n\n   >>> x = { 'a': {'b/c': 0}}\n   >>> dpath.util.get(['a', 'b/c'])\n   0\n\ndpath.path : The Undocumented Backend\n=====================================\n\ndpath.util is where\
+    \ you want to spend your time: this library has the\nfriendly functions that will understand simple string globs, afilter\nfunctions, etc.\n\ndpath.path is the backend pathing library - it is currently\n\
+    undocumented, and not meant to be used directly! It passes around lists\nof path components instead of string globs, and just generally does\nthings in a way that you (as a frontend user) might not\
+    \ expect. Stay out\nof it. You have been warned!\n\n.. |PyPI| image:: https://pypip.in/version/dpath/badge.svg?style=flat\n    :target: https://pypi.python.org/pypi/dpath/\n    :alt: PyPI: Latest Version\n\
+    \n.. |Build Status| image:: https://travis-ci.org/akesterson/dpath-python.svg?branch=travisci\n   :target: https://travis-ci.org/akesterson/dpath-python"
+  doc_url: ''
```

Comment:
Just delete this line if there's not anything meaningful to put in it, or put https://github.com/akesterson/dpath-python since that readme seems to be where the docs are.